plugins/carbon_black_defense/komand_carbon_black_defense/actions/find_event/schema.py

```diff
@@ -8,1201 +8,483 @@ class Component:
 
 
 class Input:
-    APPLICATIONNAME = "applicationName"
-    EVENTTYPE = "eventType"
-    HOSTNAME = "hostName"
-    HOSTNAMEEXACT = "hostNameExact"
-    IPADDRESS = "ipAddress"
-    OWNERNAME = "ownerName"
-    OWNERNAMEEXACT = "ownerNameExact"
-    SEARCHWINDOW = "searchWindow"
-    SHA256HASH = "sha256hash"
+    DEVICE_EXTERNAL_IP = "device_external_ip"
+    DEVICE_INSTALLED_BY = "device_installed_by"
```

Comment:
remove - call with carbon black indicated that this field is not useful to search by